scripts/ci/run_black_formatting_check.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+pip3 install black
```

Comment:
is this needed here, considering that it was added to the requirements file?

Reply:
This scripts runs inside a docker container (as black can't run on Python <3.6, which is what we have on evg), so the requirements are not installed on it. This install just black and not the whole set of requirements as they are not needed here